docs/reference/search/profile.asciidoc

```diff
@@ -163,7 +163,38 @@ The API returns the following result:
             ]
           }
         ],
-        "aggregations": []
+        "aggregations": [],
+        "fetch": {
+          "type": "fetch",
+          "description": "fetch",
+          "time_in_nanos": 660555,
+          "breakdown": {
+            "next_reader": 7292,
+            "next_reader_count": 1,
+            "load_stored_fields": 299325,
+            "load_stored_fields_count": 5
+          },
+          "debug": {
+            "stored_fields": ["_id", "_routing", "_source"]
+          },
+          "children": [
+            {
+              "type": "source",
+              "description": "load _source",
+              "time_in_nanos": 20443,
+              "breakdown": {
+                "next_reader": 745,
+                "next_reader_count": 1,
+                "process": 19698,
+                "process_count": 5
+              },
+              "debug": {
+                "loaded_nested": 0,
+                "fast_path": 5
```

Comment:
The fetch phases can emit extra debugging information if they decide to do so. This has proven super useful in aggs land. I'm not sure how useful it is here, but I figured it'd be nice to be able to glance at a profile result and know if we were able to just shovel the bytes out the other side or if we had to parse the source.